paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -171,6 +171,12 @@ public class CoreOptions implements Serializable {
                     .withDescription(
                             "Default file compression zstd level. For higher compression rates, it can be configured to 9, but the read and write speed will significantly decrease.");
 
+    public static final ConfigOption<String> FILE_PREFIX =
```

Comment:
Can you also introduce `changelog-file.prefix`?

Reply:
done @JingsongLi